enhancements/microshift/y-2-upgrades.md

```diff
@@ -0,0 +1,223 @@
+---
+title: y-2-upgrades
```

Comment:
The "-"  terminology I most often how we describe upgrades when testing. When considering a proposed release candidate, we test it's "Y" or "Z" version can be upgraded from some previous version; hence the y-1 or y-2 in this case.

Reply:
I spelled out the "minus" in the title, since the forced syntax did hide the `-` a bit. Whether it's a + or a - depends on your perspective, so while a customer may think "I'm going forward 2 versions from Y to Y+2" from the development and test standpoint the version under test is Y, so it feels more logical to talk about upgrading from Y-2 to Y.

Reply:
I will also say that when we document this we would probably not use either arithmetic expression. https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.14/html/updating/microshift-about-updates#microshift-about-updates-checking-version-update-path_microshift-about-updates lists versions explicitly but doesn't talk about the restrictions more generally. We might say something like "MicroShift supports upgrading in place across at most 2 minor versions" and then give the specific versions as we have now.

cc @ShaunaDiaz